en/advanced/ai.md

```diff
@@ -0,0 +1,79 @@
+# AI functionality in JabRef
+
+## Chat model
+
+**Type**: enumeration
+
+**Requirements**: choose one available from combo box
+
+Chat model specifies what AI models can you use. This will differ from one provider to other. Models vary in their accuracy, knowledge of the world, context window (what amount of information can they process).
```

Comment:
This should go first, because this is kind of an introduction to the field.

Reply:
You propose to move the description above and "type" and "requirements" (except for this one) to be lower?

Reply:
Yes. Both in documentation and scientific papers, one starts with some intro text and then goes into details. I think, the type is the least important information to know when one wants to know what "Chat model" means.